homeassistant/components/androidtv/media_player.py

```diff
@@ -88,13 +89,15 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up the Android Debug Bridge entity."""
-    aftv = hass.data[DOMAIN][entry.entry_id][ANDROID_DEV]
+    aftv: AndroidTVAsync | FireTVAsync = hass.data[DOMAIN][entry.entry_id][ANDROID_DEV]
     device_class = aftv.DEVICE_CLASS
     device_type = (
         PREFIX_ANDROIDTV if device_class == DEVICE_ANDROIDTV else PREFIX_FIRETV
     )
     # CONF_NAME may be present in entry.data for configuration imported from YAML
-    device_name = entry.data.get(CONF_NAME) or f"{device_type} {entry.data[CONF_HOST]}"
+    device_name: str = (
+        entry.data.get(CONF_NAME) or f"{device_type} {entry.data[CONF_HOST]}"
```

Comment:
```suggestion
        entry.data.get(CONF_NAME, f"{device_type} {entry.data[CONF_HOST]}")
```
I know it's been here before, but this does the same :)

Reply:
Good catch, done